cookbooks/swift/recipes/source.rb

```diff
@@ -109,7 +117,7 @@
 
 execute "python-swift-install" do
   cwd "#{node['source_root']}/swift"
-  command "pip install -e .[kmip_keymaster] && pip install -r test-requirements.txt"
+  command "pip install #{if not node['use_python3'] then '-c py2-constraints.txt' end} -e .[kmip_keymaster] -r test-requirements.txt"
```

Comment:
I guess that `#{ if ... end }` is just variable substitution, and the else condition is something like an implicit empty string?

```
irb(main):001:0> my_bool = true
=> true
irb(main):002:0> "test #{ if not my_bool then 'foo' end } check"
=> "test  check"
irb(main):003:0> my_bool = False
(irb):3:in `<main>': uninitialized constant False (NameError)
	from /usr/lib/ruby/gems/3.0.0/gems/irb-1.3.5/exe/irb:11:in `<top (required)>'
	from /usr/bin/irb:23:in `load'
	from /usr/bin/irb:23:in `<main>'
irb(main):004:0> my_bool = false
=> false
irb(main):005:0> "test #{ if not my_bool then 'foo' end } check"
=> "test foo check"
```

Ruby is so weird.